float/mod_test.mbt

```diff
@@ -0,0 +1,27 @@
+// Copyright 2024 International Digital Economy Academy
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+///|
+fn check(x : Float, y : Float) -> Bool {
+  (x - y).abs() < 1.0e-4
```

Comment:
Precision to 0.0001? I doubt it's too big.

Reply:
My fault, updated.